src/external.rs

```diff
@@ -26,7 +27,25 @@ impl External {
     .unwrap()
   }
 
+  pub fn new_rc<'s, T>(
+    scope: &mut HandleScope<'s, ()>,
+    value: Rc<T>,
+  ) -> Local<'s, Self> {
+    let value_ptr = Rc::into_raw(value) as *mut c_void;
+    unsafe {
+      scope.cast_local(|sd| v8__External__New(sd.get_isolate_ptr(), value_ptr))
+    }
+    .unwrap()
+  }
```

Comment:
rusty_v8 is a fairly basic, low-level binding to the V8 C++ API so this kind of abstraction arguably has no place here.